.factory/automation.yml

```diff
@@ -221,8 +221,8 @@ release:
       dependencies: [deploy-github]
       command: |
         docker login -u $REPO_DOCKER_USERNAME -p $REPO_DOCKER_PASSWORD
-        bazel run //:deploy-docker-latest
-        bazel run //:deploy-docker-versioned
+        bazel run //:deploy-docker-release
+        bazel run //:deploy-docker-release-overwrite-latest-tag
```

Comment:
match agreed upon naming conventions in typedb-cluster
- overwrite latest tag is verbose but also correctly reflects what it does on docker hub